packages/protocol/contracts/authorizers/Api3ClientRrpAuthorizer.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.6;
+
+import "./MetaAdminnable.sol";
+import "./ClientRrpAuthorizer.sol";
+import "./interfaces/IApi3ClientRrpAuthorizer.sol";
+
+/// @title Authorizer contract that whitelists clients where the API3 DAO is
+/// the metaAdmin
+contract Api3ClientRrpAuthorizer is
+    ClientRrpAuthorizer,
+    MetaAdminnable,
+    IApi3ClientRrpAuthorizer
+{
+    /// @notice Authorizer contracts use `authorizerType` to signal their type
+    uint256 public constant override AUTHORIZER_TYPE = 2;
```

Comment:
I seemed to keep forgetting what's the use case for the AUTHORIZER_TYPE value and does this value map to some enum or string constant somewhere?

Reply:
I'm guessing since this is public variable, that the node will call the contract to fetch this variable to indicate which type of authorizer it is.

Reply:
It's not really needed tbh, maybe we should remove it

Reply:
(decided to be kept)

Reply:
You are right, I was reading a medium post which was using an older version of solidity as reference.